docs/quickstart.rst

```diff
@@ -1,6 +1,166 @@
+
+    $ docker run --detach --rm -p 5432:5432 --name septentrion_db postgres
+
+.. note::
+
+    If you need to stop the docker at some point, use ``docker stop septentrion_db``.
+
+You'll also need ``psycopg2``, which is notoriously complex to install. Procrastinate
+will install the ``psycopg2`` python package, but will expect the system to already
+have the prerequisites (on ``Ubuntu``)::
+
+    $ sudo apt install libpq-dev python-dev postgresql-client
+
+.. note::
+    On a different OS, if you experiment difficulties, we'll be grateful if you can tell
+    us via an issue so that we improve this documentation.
+
+Install and configure Septentrion
+---------------------------------
+
+Within a virtualenv_, install Septentrion with:
+
+.. _virtualenv: https://packaging.python.org/tutorials/installing-packages/#creating-virtual-environments
+
+.. code-block:: console
+
+    (venv) $ pip install septentrion
+
+Next we will configure the connection to the PostgreSQL database. We can do this either with command line flags,
+environment variables or a configuration file.
+In this tutorial, we will use a configuration file, ``septentrion.ini``.
+
+.. code-block:: ini
+
+    [septentrion]
+    host=localhost
+    # port=5432
+    # username=postgres
+
+.. note::
+
+    With the Docker setup described above, you should be good to go.
+    If you need additional configuration parameters to connect to your database, have a look at
+    :ref:`advanced configuration options <how-to/configure>`.
+
+
+Write migrations
+----------------
+
+Migrations are SQL files that are executed in order. Migrations are grouped together in successive versions.
+All migrations in the same version are placed in a folder named after the version number.
+
+Let's create our migration folder, and a first version folder.
+
+.. code-block:: console
+
+    $ mkdir migrations
+    $ mkdir migrations/1.0
+
+Now we can add a migration file in ``1.0/`` named ``00-author.ddl.sql``:
+
+.. code-block:: sql
+
+    BEGIN;
+    CREATE TABLE "author" (
+        "id" serial NOT NULL PRIMARY KEY,
+        "name" varchar(100) NOT NULL,
+        "date_of_birth" varchar(10) NOT NULL
+    );
+    COMMIT;
+
+.. note::
+
+    Migrations are executed in alphabetical order, so an easy way to control the order is to prefix filenames with two
+    digits.
+
+.. note::
+
+    ``ddl`` stands for *Data Definition Language* and corresponds to all the operations that change the structure of the
+    database (``CREATE``, ``DROP``, ...).
+
+Congratulations, you have now written your first *Septentrion* migration. Let's see how to run it!
+
+
+Run migrations
+--------------
+
+First, we want to visualize what is going to happen, without making any change to our data yet.
+
+.. code-block:: console
+
+    $ septentrion --target-version 1.0 show-migrations
+
+#TODO print expected output
+
+Great, we can now run it for real:
+
+.. code-block:: console
+
+    $ septentrion --target-version 1.0 migrate
```

Comment:
We currently can't run the first migration wthout a schema and a version. And the schema isn't visible in the `show-migrations` output :-(

Reply:
Related issue for discussion https://github.com/peopledoc/septentrion/issues/54